packages/dx-react-grid/src/plugins/integrated-filtering.jsx

```diff
@@ -3,10 +3,6 @@ import * as PropTypes from 'prop-types';
 import { Getter, Plugin } from '@devexpress/dx-react-core';
 import { filteredRows, getColumnExtension } from '@devexpress/dx-grid-core';
 
-const pluginDependencies = [
-  { name: 'FilteringState' },
```

Comment:
Let's revert it and change to optional. Don't miss to add an optional dependency on SearchState